daml-lf/transaction/src/stable/protobuf/com/daml/lf/transaction.proto

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
definitely not for this PR but I think it would be great to have a `RELEASE-LF.md` file somewhere which describes the steps to go through for making a stable LF release.